barretenberg/cpp/src/barretenberg/ultra_honk/oink_verifier.cpp

```diff
@@ -17,10 +17,9 @@ template <IsUltraFlavor Flavor> OinkOutput<Flavor> OinkVerifier<Flavor>::verify(
     execute_log_derivative_inverse_round();
     execute_grand_product_computation_round();
 
-    return OinkOutput<Flavor>{
-        .relation_parameters = relation_parameters,
-        .commitments = witness_comms,
-    };
+    return OinkOutput<Flavor>{ .relation_parameters = std::move(relation_parameters),
+                               .commitments = std::move(witness_comms),
+                               .public_inputs = std::move(public_inputs) };
```

Comment:
same here, don't know if these moves are effective

Reply:
i think no point here, the data structures are not big